weave-js/src/components/PagePanelComponents/Home/Browse3/inference/modelsFinal.json

```diff
@@ -97,87 +121,20 @@
-      "downloadsHuggingFace": 2647737
-    },
-    {
-      "provider": "coreweave",
-      "id": "cw_ibm-granite_granite-3.3-8b-instruct",
-      "idPlayground": "ibm-granite/granite-3.3-8b-instruct",
-      "idHuggingFace": "ibm-granite/granite-3.3-8b-instruct",
-      "label": "Granite 3.3 8B",
-      "status": "hosted",
-      "descriptionShort": "Granite is an open, generalist model with a strong focus on reasoning, designed for transparency and broad utility across domains.",
-      "descriptionMedium": "Granite 3.3 8B represents IBM's commitment to open, transparent AI development, offering a versatile language model with strong reasoning capabilities. The model's architecture emphasizes interpretability and reliability, making it particularly suitable for enterprise applications where transparency is crucial. Despite its compact 8B parameter size, it delivers robust performance across a wide range of tasks, with particular strength in logical reasoning and structured problem-solving. The model's design prioritizes practical utility while maintaining high standards of accuracy and reliability, making it well-suited for business applications, research, and development of specialized AI solutions.",
-      "launchDate": "2025-04-01T00:00:00Z",
-      "contextWindow": 128000,
-      "priceCentsPerBillionTokensInput": 23000,
-      "priceCentsPerBillionTokensOutput": 139000,
-      "apiStyle": "chat",
-      "modalities": ["Text"],
-      "likesHuggingFace": 81,
-      "downloadsHuggingFace": 52663
-    },
-    {
-      "provider": "coreweave",
-      "id": "cw_BAAI_bge-m3",
-      "idHuggingFace": "BAAI/bge-m3",
-      "label": "BGE M3",
-      "status": "hosted",
-      "descriptionShort": "A state-of-the-art embedding model tailored for retrieval-augmented generation (RAG), providing dense, high-quality vector outputs.",
-      "descriptionMedium": "BGE M3 represents a significant advancement in embedding technology, specifically optimized for retrieval-augmented generation (RAG) applications. The model's architecture is designed to produce high-quality, dense vector representations that capture semantic meaning with exceptional precision. Its specialized training process enables it to excel at tasks requiring semantic similarity matching, information retrieval, and knowledge grounding. The model's outputs are particularly well-suited for building efficient search systems, knowledge bases, and RAG pipelines, where accurate semantic understanding is crucial for performance.",
-      "launchDate": "2024-01-01T00:00:00Z",
-      "contextWindow": 8000,
-      "priceCentsPerBillionTokensInput": 24000,
-      "priceCentsPerBillionTokensOutput": 139000,
-      "apiStyle": "embedding",
-      "modalities": ["Embedding"],
-      "likesHuggingFace": 2084,
-      "downloadsHuggingFace": 3590321
+      "modalities": [
+        "Text"
+      ],
+      "likesHuggingFace": 501,
+      "downloadsHuggingFace": 255012,
+      "license": "mit"
     }
   ]
-}
```

Comment:
should this have a newline?

Reply:
Won't break anything without but I've fixed it